chia/full_node/full_node.py

```diff
@@ -176,9 +176,7 @@ def sql_trace_callback(req: str):
 
         db_version: int = await lookup_db_version(self.connection)
 
-        self.db_wrapper = DBWrapper(
-            self.connection, self.config.get("allow_database_upgrades", False), db_version=db_version
-        )
+        self.db_wrapper = DBWrapper(self.connection, db_version=db_version)
```

Comment:
Is this a correct change?  I thought the "allow_database_upgrades" option was new and as such should probably be in here.

Reply:
Removed in https://github.com/Chia-Network/chia-blockchain/pull/9768.